src/pubkey.h

```diff
@@ -205,4 +210,13 @@ struct CExtPubKey {
     bool Derive(CExtPubKey& out, unsigned int nChild) const;
 };
 
+/** Users of this module must hold an ECCVerifyHandle. The constructor and
+ *  destructor of these are not allowed to run in parallel, though. */
```

Comment:
@laanwj wrote:
> Users have to hold a ECCVerifyHandle. Maybe we should make them pass it in to make sure of that? :-)

I agree. It may look weird because it doesn't appear to be used.
But if secp256k1_context_verify and verify_refcount were attributes of ECCVerifyHandle, we would see much clearly which functions/methods need to take the ECCVerifyHandle as parameter.
I suggest doing both changes.